app/components/form-block-radiobutton/component.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    classNames: ['osf-box'],
+
+    title: null,
+    description: null,
+    value: null,
+    // Namespace for radiobuttons
+    name: null,
+
+    // Bound attribute
+    attribute: null,
+
+    checked: Ember.computed('attribute', function() {
```

Comment:
Should depend on `'value'`, too, in case someone ever tries to use this with mutable value.